LWCOSC/force-app/main/default/lwc/productListNew/productListNew.js

```diff
@@ -0,0 +1,186 @@
+import { LightningElement, api, wire, track } from 'lwc';
+
+
+import getProducts from '@salesforce/apex/ProductController.getProducts';
+
+import NAME_FIELD from '@salesforce/schema/Product2.Name';
+import PRICE_FIELD from '@salesforce/schema/Product2.Total_Price__c';
+import PRODUCT_CODE_FIELD from '@salesforce/schema/Product2.ProductCode';
+import AVAILABLE_UNITS_FIELD from '@salesforce/schema/Product2.Available_Quantity__c';
+
+
+const PRODLISTCOLUMNS = [
+    { label: 'Name', fieldName: NAME_FIELD.fieldApiName, type: 'text', sortable: true },
+    { label: 'Price', fieldName: PRICE_FIELD.fieldApiName, type: 'currency', sortable: true },
+    { label: 'Product Code', fieldName: PRODUCT_CODE_FIELD.fieldApiName, type: 'text', sortable: true },
+    { label: 'Available Units', fieldName: AVAILABLE_UNITS_FIELD.fieldApiName, type: 'number', sortable: true }
+];
+
+export default class ProductListNew extends LightningElement {
+    productListFlag = true;
+    hideCheckProductListTable = false;
+	errors=[];
+	selectedProductList = [];
+	selectedRows = [];
+	isGoToCartBTNDisabled = false;
+	searchKey = '';
+    setSelectedRows;
+    @api sortedDirection = 'asc';
```

Comment:
All properties with similar decorators/types should be placed in groups for better readability. E.g. all properties with "@api" should be together, all "@track" should be together, all handlers should be together etc.

Reply:
Done